torch/utils/model_zoo.py

```diff
@@ -125,7 +125,7 @@ def _download_url_to_file(url, dst, hash_prefix, progress):
     # fake tqdm if it's not installed
     class tqdm(object):
 
-        def __init__(self, total, disable=False):
+        def __init__(self, total=None, disable=False):
```

Comment:
Nice catch.

cc: @fmassa

Reply:
I know, this was the original reason for the breakage in torchvision :-)
I worked around it, but having this default constructor was something I was considering doing as well